### DISCUSSION TITLE
Compiling help: cannot open include file 'crtdbg.h' no such file or directory

### DISCUSSION THREAD

**Discussion by YousefTamer67** [FunkinCrew/Funkin#1544] [Compiling help]
## What is your question?
![Screenshot (156)](https://user-images.githubusercontent.com/82932252/125203649-5dae4b80-e28a-11eb-81db-846be6c17390.png)
i can't compile it i am using visual studio build tools

**Comment by ghost** [comment]
Make sure you installed everything

**Comment by YousefTamer67** [comment]
i did

**Comment by ghost** [comment]
> i did

follow the readme

**Comment by YousefTamer67** [comment]
i did install everything

**Comment by YousefTamer67** [comment]
but something just say that no such file
